Extension/src/LanguageServer/configurations.ts

```diff
@@ -206,7 +206,7 @@ export class CppProperties {
 
             if (!settings.defaultIncludePath) {
                 // We don't add system includes to the includePath anymore. The language server has this information.
-                configuration.includePath = ["${workspaceFolder}/**"].concat(this.vcpkgIncludes);
+                configuration.includePath = ["${workspaceFolder}"].concat(this.vcpkgIncludes);
```

Comment:
nit: `this.vcpkgIncludes.unshift("${workspaceFolder}")`

Reply:
My understanding is that unshift modifies the source array and returns the length instead of a new array.  This is not what we want here.